Utils/Dataflow/pyDKB/storages/atlas/rucio.py

```diff
@@ -0,0 +1,110 @@
+"""
+pyDKB.storages.atlas.rucio
+"""
+
+import os
+
+from ..client import Client
+from ..exceptions import (StorageException, NotFound)
+from pyDKB.common.misc import (log, logLevel)
+from pyDKB.common.misc import try_to_import
+
+
+if not os.environ.get("VIRTUAL_ENV", None):
+    user_rucio_dir = os.path.expanduser("~/.rucio")
+    if os.path.exists(user_rucio_dir):
+        os.environ["VIRTUAL_ENV"] = os.path.join(user_rucio_dir)
+    else:
+        os.environ["VIRTUAL_ENV"] = os.path.join(base_dir, ".rucio")
```

Comment:
`base_dir` is not defined. Forgot to copy
https://github.com/PanDAWMS/dkb/blob/master/Utils/Dataflow/091_datasetsRucio/datasets_processing.py#L15
?

Reply:
Good point, fixed now.